build-tools/packages/build-tools/src/fluidBuild/fluidPackageCheck.ts

```diff
@@ -149,10 +149,30 @@ export class FluidPackageCheck {
     ) {
         let fixed = false;
         const actual = pkg.getScript(name);
+
         if (expected !== actual) {
-            this.logWarn(pkg, `non-conformant script "${name}"`, fix);
-            this.logWarn(pkg, `  expect: ${expected}`, fix);
-            this.logWarn(pkg, `  actual: ${actual}`, fix);
+            if (name === "lint" || name === "lint:fix") {
+                /* Regular Expression
+                    lintRegex is a regex to extract expected lint-packages from the expected string value
+
+                    if expected = "npm run foo && npm run bar && npm run baz-qux:fix"
+                    lintPackages = ["foo", "bar", "baz-qux"]
+
+                    `/run (.*?)(?= &&|:|$)/g` : matches text between "run" and ("&&" or ":")
+                    `map` : extract the text from lintMatch by replacing `run` and `\s` with ""
+                */
+
+                const lintRegex = /run (.*?)(?= &&|:|$)/g;
+                const lintMatch = expected?.match(lintRegex);
```

Comment:
Shouldn't we be running the match against `actual`, rather than `expected`?

Reply:
I ran it against `expected` as we would like to log packages that must be included in the script. When passing the `lintChildren` and `lintFixChildren` to `checkChildrenScripts()`, `eslint` is appended to each array if it is not included in the script. Therefore, `lintChildren` and `lintFixChildren` become the expected script. 

After completing the steps above, our error log will run regex against the `expected` string variable and separate values based on its package name (discard `npm run` `:fix` `&&`) and log the name of packages on console.